clusters/nerc-ocp-infra/xdmod/application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: xdmod
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/CCI-MOC/xdmod-cntr.git
+    targetRevision: HEAD
+    path: k8s/overlays/nerc-ocp-infra
```

Comment:
Your patch at https://github.com/CCI-MOC/xdmod-cntr/pull/64 refers to the overlay as being called `NERC` rather than `nerc-ocp-infra`. Either rename it here or there to match.

Reply:
I actually like this naming convention better than NERC, so I changed it here (https://github.com/CCI-MOC/xdmod-cntr/pull/64).